### ISSUE TITLE
Is it possible to use this as a loader?

### ISSUE THREAD

**Issue by themindstorm** [eemeli/yaml#141]
In webpack.config.js, something similar to this:
```
config.module.rules.push({
  test: /\.yml$/,
  use: 'yaml',
})
```

I'm asking because I'm wondering if it's possible to import a file as a JS object like so:
```
const jsObject = import("./path/to/file.yml")
```

**Comment by eemeli** [comment]
Not yet, but let me get back to you on this sometime soon.

**Comment by themindstorm** [comment]
Okay, thanks!

**Comment by eemeli** [comment]
I've just published [`yaml-loader@0.6.0`](https://www.npmjs.com/package/yaml-loader) on npm, which is now a loader using this library, with support for all its options. The package's original author @okonet kindly allowed me to take over its maintenance.

Please note that its 0.6.0 release is intended to be backwards-compatible to its earlier API, which means that its results are stringified JSON rather than JS. This means that you'll need to include a `type: 'json'` specifier for the Webpack config rule, like this:

```js
{
  test: /\.ya?ml$/,
  type: 'json',
  use: 'yaml-loader'
}
```

**Comment by eemeli** [comment]
Closing as answered.
